timemachine/fe/single_topology.py

```diff
@@ -1040,7 +1040,7 @@ def assert_bonds_defined_for_chiral_volumes(
                 raise MissingBondsInChiralVolumeException(f"bond {(c, k)} missing from Chiral Volume {(c, i, j, k)}")
 
 
-def assert_torsions_defined_over_non_linear_angles(system: VacuumSystem | HostGuestSystem):
+def assert_torsions_defined_over_non_linear_angles(system: GuestSystem | HostGuestSystem | HostSystem):
```

Comment:
nit: it feels more precise for this to take `angle`, `proper`, and `improper` as arguments (then it could work with future system containers with these potentials too)

Reply:
yes - but lets defer it, I have to update quite a number of tests to do this properly.